qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -544,21 +550,27 @@ def convert_set_frequency(self, instruction):
 
         Args:
             instruction (PulseQobjInstruction): set frequency qobj instruction
+                                                The passed frequency will be scaled by a factor 1e9
```

Comment:
On second thought, maybe it would be better to say something like: "Frequency is intended in (GHz)"?